eventually/task/models.py

```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
     status = models.IntegerField(default=0, choices=STATUS_TYPE_CHOICES)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
+    users = models.ManyToManyField(CustomUser)
+
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        task instance object.
+
+        :return: task title, task description, task status
+        """
 
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,

```

Comment:
you have event = models.ForeignKey(CustomUser, null=True)
so add self.event.id if self.owner else None,

Reply:
I have  event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
owner??? 
there is no such field in my model :)